.babelrc

```diff
@@ -19,10 +19,12 @@
                     "@": "./",
                     "@server": "./server",
                     "@client": "./client",
+                    "@components": "./client/pages/Home/Components",
```

Comment:
The `@components` alias should point to `./client/components`

Reply:
resolved

Reply:
No, not resolved.